jenkins/linux_bearssl.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+set -e
+
+# Print version
+cat /etc/*release | grep VERSION*
+gcc --version
+curl --version
```

Comment:
Can you print the bearssl version?

Reply:
Not easily, it isn't a self contained tool, it's a library.